packages/web3-eth-contract/src/types.ts

```diff
@@ -376,7 +376,7 @@ export interface PayableMethodObject<Inputs = unknown[], Outputs = unknown[]> {
 	 * @returns - The return value(s) of the smart contract method. If it returns a single value, it’s returned as is. If it has multiple return values they are returned as an object with properties and indices.
 	 */
 	call<SpecialOutput = Outputs>(
-		tx?: PayableCallOptions,
+		tx?: Omit<PayableCallOptions, 'nonce'>,
```

Comment:
This omission should be removed as well, no? Same for `estimateGas` and `createAccessList` below